examples/sendToAndTransferFromStealthAddress/components/stealth-actions-example.tsx

```diff
@@ -0,0 +1,255 @@
+  if (!rpcUrl) throw new Error('VITE_RPC_URL is required');
+  if (!window.ethereum) throw new Error('window.ethereum is required');
+
+  const CHAIN = sepolia;
+  const SCHEME_ID = VALID_SCHEME_ID.SCHEME_ID_1;
+  const MESSAGE_TO_SIGN = `Generate Stealth Meta-Address on ${CHAIN.id} chain`;
+  const SEND_AMOUNT = '0.01';
+  const WITHDRAW_AMOUNT = '0.005';
+
+  const walletClient = createWalletClient({
+    chain: CHAIN,
+    transport: custom(window.ethereum)
+  });
+
+  const [connectedAccount, setConnectedAccount] = useState<Address>();
+  const [keys, setKeys] = useState<{
+    spendingPublicKey: `0x${string}`;
+    spendingPrivateKey: `0x${string}`;
+    viewingPublicKey: `0x${string}`;
+    viewingPrivateKey: `0x${string}`;
+  }>();
+  const [stealthAddressDetails, setStealthAddressDetails] =
+    useState<GenerateStealthAddressReturnType>();
+  const [sendToStealthTxHash, setSendToStealthTxHash] =
+    useState<`0x${string}`>();
+  const [transferFromStealthTxHash, setTransferFromStealthTxHash] =
+    useState<`0x${string}`>();
+  const [currentStep, setCurrentStep] = useState(0);
+  const [error, setError] = useState<string | null>(null);
+
+  const {
+    isLoading: isLoadingSendToStealthTx,
+    isSuccess: isSuccessSendToStealthTx
+  } = useWaitForTransactionReceipt({ hash: sendToStealthTxHash });
+  const {
+    isLoading: isLoadingTransferFromStealthTx,
+    isSuccess: isSuccessTransferFromStealthTx
+  } = useWaitForTransactionReceipt({ hash: transferFromStealthTxHash });
+
+  const {
+    data: connectedAccountBalance,
+    isLoading: isConnectedAccountBalanceLoading,
+    refetch: refetchConnectedAccountBalance
+  } = useBalance({
+    address: connectedAccount
+  });
+
+  const {
+    data: stealthAddressBalance,
+    isLoading: isStealthAddressBalanceLoading,
+    refetch: refetchStealthAddressBalance
+  } = useBalance({
+    address: stealthAddressDetails?.stealthAddress
+  });
+
+  const connect = async () => {
+    try {
+      const [address] = await walletClient.requestAddresses();
+      setConnectedAccount(address);
+      setCurrentStep(1);
+    } catch (error) {
+      handleError('Error connecting wallet');
+    }
+  };
+
+  const getSignature = async ({ message }: { message: string }) => {
+    if (!connectedAccount) throw new Error('A connected account is required');
+    return await walletClient.signMessage({
+      message,
+      account: connectedAccount
+    });
+  };
+
+  const getStealthAddressDetails = async () => {
+    try {
+      const signature = await getSignature({ message: MESSAGE_TO_SIGN });
+      const newKeys = generateKeysFromSignature(signature);
+      setKeys(newKeys);
+      const stealthMetaAddress =
+        generateStealthMetaAddressFromSignature(signature);
+      const details = generateStealthAddress({
+        stealthMetaAddressURI: stealthMetaAddress,
+        schemeId: SCHEME_ID
+      });
+      setStealthAddressDetails(details);
+      setCurrentStep(2);
```

Comment:
instead of using these numbers, can we use enums instead

Reply:
good call:

https://github.com/ScopeLift/stealth-address-sdk/pull/77/commits/0f9bc0e166cfed0b8386fe3fc3f1cb962f47bcf9